triton-server/src/download/schema.sql

```diff
@@ -46,4 +46,11 @@ create table contacts (
     type       text check(type IN ('SFTP', 'GLOBUS')) not null,
 
    unique(project_id, type)
-);
\ No newline at end of file
+);
+
+create table constants (
+    id                  INTEGER PRIMARY KEY ASC check(id = 1),
```

Comment:
You can remove the id if you can ...

Reply:
Nah. It's very handy to avoid inserting new ;)